deploy/openshift/clusterrole.yaml

```diff
@@ -74,63 +48,92 @@ rules:
   - stackdrivers
   - statsds
   - stdios
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - watch
+  verbs: ["create", "delete", "get", "list", "patch", "watch"]
 - apiGroups: ["networking.istio.io"]
-  resources:
-  - destinationrules
-  - gateways
-  - serviceentries
-  - virtualservices
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - watch
+  resources: ["destinationrules", "gateways", "serviceentries", "virtualservices"]
+  verbs: ["create", "delete", "get", "list", "patch", "watch"]
 - apiGroups: ["authentication.istio.io"]
-  resources:
-  - policies
-  - meshpolicies
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - watch
+  resources: ["policies", "meshpolicies"]
+  verbs: ["create", "delete", "get", "list", "patch", "watch"]
 - apiGroups: ["rbac.istio.io"]
-  resources:
-  - clusterrbacconfigs
-  - serviceroles
-  - servicerolebindings
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - watch
+  resources: ["clusterrbacconfigs", "serviceroles", "servicerolebindings"]
+  verbs: ["create", "delete", "get", "list", "patch", "watch"]
 - apiGroups: ["apps.openshift.io"]
-  resources:
-  - deploymentconfigs
-  verbs:
-  - get
-  - list
-  - watch
+  resources: ["deploymentconfigs"]
+  verbs: ["get", "list", "watch"]
 - apiGroups: ["project.openshift.io"]
-  resources:
-  - projects
-  verbs:
-  - get
+  resources: ["projects"]
+  verbs: ["get"]
 - apiGroups: ["monitoring.kiali.io"]
-  resources:
-  - monitoringdashboards
-  verbs:
-  - get
+  resources: ["monitoringdashboards"]
+  verbs: ["get"]
+---
+apiVersion: v1
+kind: ClusterRole
+metadata:
+  name: kiali-viewer
+  labels:
+    app: kiali
+    version: ${VERSION_LABEL}
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps", "endpoints", "namespaces", "nodes", "pods", "services", "replicationcontrollers", "routes"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["extensions", "apps"]
+    resources: ["deployments", "statefulsets", "replicasets"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["autoscaling"]
+    resources: ["horizontalpodautoscalers"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["batch"]
+    resources: ["cronjobs", "jobs"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["config.istio.io"]
+    resources:
```

Comment:
This section is what I mean by different syntax being used. This list uses "one item per line" but other places its putting all the items on a single line.

I do not like the "all itesm on a single line" because its hard to do a diff visually. I like seeing one item per line (with the items alphabetized) because it makes it easier to find something if I'm looking for it.